packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -459,14 +457,17 @@ export const useDamFileUpload = (options: UploadDamFileOptions): FileUploadApi =
                         } else {
                             addValidationError(file, <UnknownError />);
                         }
-                    } else if (hasStringErrorData(typedErr) && typedErr.response?.data.includes("SVG contains forbidden content")) {
+                    } else if (
+                        hasStringErrorData(typedErr) &&
+                        typeof typedErr.response?.data === "string" &&
```

Comment:
This additional type check shouldn't be necessary if hasStringErrorData is correctly typed.